cachito/workers/pkg_managers/gomod.py

```diff
@@ -93,6 +94,42 @@ class GoPackage(_GolangModel):
     deps: list[str] = []
 
 
+class LocalModules(pydantic.BaseModel):
```

Comment:
IMO there's no reason to make this a pydantic model, the class doesn't represent any external data structure

A dataclass would work just fine